config/oidc.yml.docker

```diff
@@ -0,0 +1,10 @@
+test: &default
```

Comment:
this file looks to have the same content as `config/oidc.yml`, could we use that instead?